### ISSUE TITLE
How to solve runserver.py error: argument -k/--gmaps-key

### ISSUE THREAD

**Issue by oydfcool** [AHAAAAAAA/PokemonGo-Map#2563]
## Steps to Reproduce (for bugs)

1.
2.
3.
4.

## Your Environment

* Version used:
* Environment name and version (e.g. Python 2.7):
* Operating System and version (desktop or mobile):

**Comment by Plop** [comment]
You just have to read the error : the -k argument is missing. So add it.

**Comment by Dragonprovidence** [comment]
If you were planning to run it without arguments, then you'll still at least need to supply the minimum settings in the config.ini file.

**Comment by oydfcool** [comment]
My command now is: C:\Users\lenovo\Downloads\PokemonGO-Map-master>C:\Python27\python.exe runserver.py -ar 10 -u ****** -p ****** -l "location" -st 10

It shows: runserver.py: error: argument -k/--gmaps-key is required.

**Comment by willgrnr** [comment]
It is literally telling you what you need to do. Add that flag with your API key.

**Comment by Dragonprovidence** [comment]
Yes, as others pointed out, it is difficult to get more clear on the explanations. Have you read over the installation guide on the wiki? https://www.jonaharagon.com/PoGoMapWiki/#!installation.md

**Comment by Plop** [comment]
How can you not understand the "argument -k IS REQUIRED" ? Just add it.

**Comment by mortasgus** [comment]
what is "required" ? :-)

mandatory maybe ? ^_^

**Comment by TJohnson93** [comment]
Like others have said... add the -k argument (Google Maps API Key) to your command:

`runserver.py -ar 10 -u ****** -p ****** -l "location" -k [GMAPS API KEY] -st 10`

or add your GMAPS API KEY to the **config.ini** file found in the **config** folder..... Also have a read through the [Installation Instructions](https://www.jonaharagon.com/PoGoMapWiki/#!installation.md) for instructions to install and how to generate a [Gmaps API Key](https://www.jonaharagon.com/PoGoMapWiki/#!GoogleMaps.md)

**Comment by oydfcool** [comment]
Thanks all!!! I made it!!

**Comment by commanderjsoc** [comment]
You can also just put the key in config.ini @oydfcool
